### PR TITLE
fix(combobox): chips vertical alignment.

### PR DESCRIPTION
**Pull request by juliodialpad**
# Fix combobox multi-select chips vertical alignment.

## :hammer_and_wrench: Type Of Change

- [x] Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation

## :book: Description

Added constants with needed inputWrapper paddings to correctly align items

## :bulb: Context

Recent Input component CSS updates changed size and spacing, impacting the Vue Combobox recipes' vertical alignment.

## :pencil: Checklist

- [x] I have reviewed my changes
- [ ] I have added tests
- [ ] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :camera: Screenshots / GIFs
**md**
image

**sm**
image

**xs**
image

### CODE REVIEW

**Review by francisrupert — COMMENTED** [review]
Looks a heckuva lot better. View the attached screencast, noting how input height changes once item selected. Impacts all three sizes.

Ensure the input matches the height of the [base Input sizes](https://dialpad.design/components/input.html#input-sizes).

https://user-images.githubusercontent.com/1165933/213287740-eae6a354-b984-41a0-8b84-a40d6ca4c1f2.mov

**Comment by juliodialpad** [comment]
> Looks a heckuva lot better. View the attached screencast, noting how input height changes once item selected. Impacts all three sizes.
> 
> Ensure the input matches the height of the [base Input sizes](https://dialpad.design/components/input.html#input-sizes).
> 
>  combobox-height.mov

The size change is intended, the badge sizes are bigger than the input field, that's why we need to add some padding to the bottom, If I don't add any padding, the badge element looks bigger than the actual input.

I can add a default padding to the input so the change is no noticeable, but I don't know if that is a conflict too.

**Comment by braddialpad** [comment]
Hmm wonder if we can use smaller size chips here?

**Comment by juliodialpad** [comment]
> Hmm wonder if we can use smaller size chips here?

`chip` size is equal to `input` size, I'll test clipping chip to 'md' size max to check if that solves the issue.

**Comment by braddialpad** [comment]
How does it look if we use chip one size smaller than input size?

**Comment by juliodialpad** [comment]
> How does it look if we use chip one size smaller than input size?

It's better but still need to remove the padding and find another way to center the chips, working on a fix.

File: recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue [hunk 1]
```diff
@@ -361,23 +368,36 @@ export default {
...
       // Adjust the chips position if label changed
-      this.setChipsTopPosition();
+      this.setChipsPosition();
     },
 
     async description () {
       await this.$nextTick();
       // Adjust the chips position if description changed
-      this.setChipsTopPosition();
+      this.setChipsPosition();
```
Comment (francisrupert) [inline recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue:377]: Really like the math'ing update you got here. Very readable, and I can see exactly what's going on.

**Review by francisrupert — APPROVED** [review]
Great update. Good call on Chip size adjustment. We've talked about the need for 3 different sizes, and this will inform that conversation.

**Review by braddialpad — APPROVED** [review]
Looks far better, thanks

**Pull request merged**